safegraph/tests/test_process.py

```diff
@@ -62,5 +70,112 @@ def test_handle_wip_signal(self):
+            'x/y/z/2020/07/04/2020-07-04-social-distancing.csv.gz')) ==\
+            ('x/y/z/2020/07/03/2020-07-03-social-distancing.csv.gz',
+             'x/y/z/2020/07/02/2020-07-02-social-distancing.csv.gz',
+             'x/y/z/2020/07/01/2020-07-01-social-distancing.csv.gz',
+             'x/y/z/2020/06/30/2020-06-30-social-distancing.csv.gz',
+             'x/y/z/2020/06/29/2020-06-29-social-distancing.csv.gz',
+             'x/y/z/2020/06/28/2020-06-28-social-distancing.csv.gz')
+        # Week that stretches over a year boundary.
+        assert tuple(files_in_past_week(
+            'x/y/z/2020/01/04/2020-01-04-social-distancing.csv.gz')) ==\
+            ('x/y/z/2020/01/03/2020-01-03-social-distancing.csv.gz',
+             'x/y/z/2020/01/02/2020-01-02-social-distancing.csv.gz',
+             'x/y/z/2020/01/01/2020-01-01-social-distancing.csv.gz',
+             'x/y/z/2019/12/31/2019-12-31-social-distancing.csv.gz',
+             'x/y/z/2019/12/30/2019-12-30-social-distancing.csv.gz',
+             'x/y/z/2019/12/29/2019-12-29-social-distancing.csv.gz')
+        # Week that includes a leap day.
+        assert tuple(files_in_past_week(
+            'x/y/z/2020/03/01/2020-03-01-social-distancing.csv.gz')) ==\
+            ('x/y/z/2020/02/29/2020-02-29-social-distancing.csv.gz',
+             'x/y/z/2020/02/28/2020-02-28-social-distancing.csv.gz',
+             'x/y/z/2020/02/27/2020-02-27-social-distancing.csv.gz',
+             'x/y/z/2020/02/26/2020-02-26-social-distancing.csv.gz',
+             'x/y/z/2020/02/25/2020-02-25-social-distancing.csv.gz',
+             'x/y/z/2020/02/24/2020-02-24-social-distancing.csv.gz')
+
+    def test_process_window(self, tmp_path):
+        """Tests that processing over a window correctly aggregates signals."""
+        export_dir = tmp_path / 'export'
+        export_dir.mkdir()
+        df1 = pd.DataFrame(data={
+            'date_range_start': ['2020-02-14T00:00:00-05:00:00']*3,
+            'origin_census_block_group': [10539707003,
+                                          10539707003,
+                                          10730144081],
+            'device_count': [100, 200, 1000],
+            'completely_home_device_count': [2, 12, 40]
+        })
+        df2 = pd.DataFrame(data={
+            'date_range_start': ['2020-02-14T00:00:00-05:00:00'],
+            'origin_census_block_group': [10730144081],
+            'device_count': [2000],
+            'completely_home_device_count': [480]
+        })
+        process_window([df1, df2], ['completely_home_prop'], ['county'],
+                       export_dir)
+        expected = pd.DataFrame(data={
+            'geo_id': [1053, 1073],
+            'val': [0.04, 0.14],
+            'se': [0.02, 0.10],
+            'sample_size': [2, 2]
+        })
+        actual = pd.read_csv(
+            export_dir / '2020-02-14_county_completely_home_prop.csv')
+        print(expected)
+        print(actual)
```

Comment:
are these debug statements?

Reply:
Yes, removed sorry for having the multi sync workspace here.